apps/communications/dialer/style/suggestion.css

```diff
@@ -7,6 +7,12 @@
   width: 100%;
   color: #033B37;
 }
+html[dir="ltr"] .suggestion-bar {
```

Comment:
Interesting pattern of grouping all ```[dir=***]``` keys together with their general counterparts. Our pattern so far has been to space them out more, though, so let's stick with that.